script/premint/output.csv

```diff
@@ -0,0 +1,1708 @@
+optimize,75871129395515024611414876210493065458476519828110085382677525358496343874967
+hash,23072298399147358551964600192559211694446004394106715762130910773502572810593
+sign,53475421289989775264717344269565944398992096593011799777690842313955336172207
+verify,104255699964326559459929165697374567282625242439140960210013884089192161540051
+validate,46966089806584262389315307097559820211029942242335328780862012689900616821194
+commit,76663182157542443246570966778786733915994856170286172035398161380253349551133
+mine,35060499484825575065469050923271593381876410654857279584153499131010280767341
+govern,29614972336742151279656662373713816913931918767964359406289953943511606077302
+wrap,61919776458315551670570029748789209304310150076416119489431512047584513615309
+unwrap,23916177387335054282768059132310083050140172901659915666180471634212122683160
+sync,70051314060174338749199528776036503126570365110744264978023464500691390694429
+broadcast,15778936680669811537001991550475160541163349550222777435794180063945058387492
+encrypt,114074440753071270710689404285137595353124877553010025862328058432201697752406
+issue,73834880146575938389772868482689524984795823800433129901915554777146741702013
+airdrop,110771216890900307486995680796878979552050216338441285842533727414245120540081
+forge,81853783688437246423385043888133248765190044475989032810777593796260099592799
+drop,50280025373204598438615537120241199798601461512960155762692969937545175591833
+stake,88484195063321507531630856583466605305443211162192762983291663561656807272313
+unstake,55292139899201075059694156460875689104618554755125664408086260014536070521388
+liquidate,46239033144713571295457171060548180312627718934696963441090248731831765313762
+pool,102836084429613045306211152906088615203204493681036628213737159806395275099389
+farm,25853177690240242883766006831280007669041172026396797835670157823339472153080
+bond,6862108173542606817271216367386974664856612571109159223273361063481523205561
+unbond,73549546149440153098815415118333932171920041436714270675245627139753386967920
+rebase,29686004246511378615486892759016583054499029954947746154360806117538245169698
+migrate,72986116339150446082689171759700430760106414694420992940860735748002122470944
+anchor,10763923030673128046922049100766745327740066525000459267710821538481838926965
+signal,7384510163819699480102054571688096275510944888734149155696244307170797257064
+index,71552106845550140871983922175974683529144468276811949388105619611751859457938
+flash,85558357609495216318253919252242190203722505890697218204638494070271760923298
+drain,24701708314572634357916250440134022151723757036256723111406180233023575726768
+aggregate,102330316382983605854315020259007756269403317351933302774740449722392990073844
+rodeo,47843225644331270063128493724511267146276014100898521270698446977699084126641
+coln,52542356754606411786199551460370531840268728762863954976333291853326614804026
```

Comment:
`coin` phish names are in output, but not in any of the premint files

Reply:
These names (and some others we got from the product org ad-hoc) were minted by manually executing the Premint script.